legend-engine-xt-serviceStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/to/HelperServiceStoreGrammarComposer.java

```diff
@@ -195,15 +230,25 @@ else if (typeReference instanceof ComplexTypeReference)
         return builder.toString();
     }
 
-    private static String renderAuthenticationStrategy(SecurityScheme securityScheme)
+    private static String renderSecurityScheme(String id, SecurityScheme securityScheme, PureGrammarComposerContext context)
+    {
+        List<Function3<String, SecurityScheme, PureGrammarComposerContext, String>> processors = ListIterate.flatCollect(IServiceStoreGrammarComposerExtension.getExtensions(), ext -> ext.getExtraSecuritySchemesComposers());
+
+        return ListIterate.collect(processors, processor -> processor.value(id, securityScheme, context))
```

Comment:
can we stage it similar to auth below where we don't pass id to extensions and instead of context pass baseIndent?